modules/dxp/apps/analytics/analytics-reports-js-components-web/src/main/resources/META-INF/resources/js/components/cms/TrafficChannels.tsx

```diff
@@ -52,19 +65,19 @@ const TrafficChannelsEntry = ({
 }) => {
 	return (
 		<div
-			aria-label={`${Liferay.Language.get('traffic-channel')}: ${Liferay.Language.get(name)}`}
+			aria-label={`${Liferay.Language.get('traffic-channel')}: ${Liferay.Language.get(TrafficChannelsNames['name'])}`}
```

Comment:
Change it to `${Liferay.Language.get('traffic-channel')}: ${trafficChannelsNames['name']}`